frontend/src/advising/Templates/TemplateListPage.tsx

```diff
@@ -188,7 +187,7 @@ const TemplatesList = ({ searchQuery }: TemplatesListProps) => {
   }, [searchQuery]);
 
   return (
-    <TemplatesContainer>
+    <>
```

Comment:
This is to get rid of the weird double red outline display that was happening on the TemplateListPage